### PR TITLE
telephonenumbers

### PR DESCRIPTION
**Pull request by feliciaan**

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -72,9 +72,9 @@ def generate_xls
     xls = Spreadsheet::Workbook.new
     sheet = xls.create_worksheet
 
-    sheet.update_row 0, "Naam", "Email", "Studentnummer", "Ticket", "Comment", "Te betalen"
+    sheet.update_row 0, "Naam", "Email", "Studentnummer", 'Telefoonnummer',"Ticket", "Comment", "Te betalen"
```
Comment (Silox) [inline app/models/event.rb]: Use consistent quotes

File: app/views/application/_form_telephone_field.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="form-group">
+  <%= f.label tag %>:<br/>
+  <%= f.telephone_field tag, class: 'form-control' %>
+  <small>Your telephone number will be kept confidential,...</small>
```
Comment (Silox) [inline app/views/application/_form_telephone_field.html.erb]: Is this to be completed? I propose something like: 
> Your telephone number will be kept confidential and can only be consulted by event organizers.

File: app/views/registrations/_info_content.html.erb [hunk 3]
```diff
@@ -5,6 +5,9 @@
   </p>
 <% end %>
 
+<h4>Telephonenumber</h4>
```
Comment (Silox) [inline app/views/registrations/_info_content.html.erb]: space

File: app/views/application/_form_telephone_field.html.erb (hunk 2 above)
Comment (feliciaan) [inline app/views/application/_form_telephone_field.html.erb]: Yes, the text should be changed.

**Comment by feliciaan** [comment]
The phone number is not mandatory for the moment.

File: app/models/registration.rb [hunk 4]
```diff
@@ -126,4 +128,8 @@ def to_cents(value)
     if value.is_a? String then value.sub!(',', '.') end
     (value.to_f * 100).to_int
   end
+
+  def phone_number_required?
+    return event && event.phone_number_state=='required'
```
Comment (nudded) [inline app/models/registration.rb]: spacing around the '=='.

File: app/views/registrations/_basic.html.erb [hunk 5]
```diff
@@ -7,7 +7,9 @@
     <%= form_text_field f, :name %>
     <%= form_email_field f, :email %>
     <%= f.hidden_field :student_number %>
-
+    <% unless @registration.event.phone_number_state == 'disabled' %>
```
Comment (nudded) [inline app/views/registrations/_basic.html.erb]: do not check explicitly for 'disabled' here, make it a method on the event

**Pull request merged**